From: Paul Wise <pabs@debian.org>
Date: Mon, 19 Sep 2016 23:24:24 +0200
Subject: use-system-libraries

use system libraries instead of embedded code copies
Forwarded: no

Should do something better with -lminiupnpc

Also should stop it compiling 3rdparty/miniupnp when we use the system one
---
 configure.ac    | 5 -----
 po/update-po.sh | 2 +-
 src/Makefile.am | 7 +++----
 3 files changed, 4 insertions(+), 10 deletions(-)

Index: warzone2100-4.2.1/po/update-po.sh
===================================================================
--- warzone2100-4.2.1.orig/po/update-po.sh
+++ warzone2100-4.2.1/po/update-po.sh
@@ -17,6 +17,6 @@ EOF
 
 find lib src data po -type f |
 	grep -e '\.c\(pp\|xx\)\?$' -e 'data.*strings.*\.txt$' -e 'data.*sequenceaudio.*\.tx.$' -e '\.slo$' -e '\.rmsg$' -e 'po/custom/.*\.txt' -e '\.js$' |
-	grep -v -e '\.lex\.c\(pp\|xx\)\?$' -e '\.tab\.c\(pp\|xx\)\?$' -e 'lib/netplay/miniupnpc/*' -e 'lib/betawidget/*' -e '_moc\.' -e 'po/custom/files.js' |
+	grep -v -e '\.lex\.c\(pp\|xx\)\?$' -e '\.tab\.c\(pp\|xx\)\?$' -e 'lib/betawidget/*' -e '_moc\.' -e 'po/custom/files.js' |
 	grep -v -e '_lexer\.cpp' -e '_parser\.cpp' |
 	sort >> po/POTFILES.in
